packages/credentials/lib/credentials.test.js

```diff
@@ -100,5 +100,18 @@ describe('Credentials module', () => {
       const error = await getPromiseError(() => credentials.addFromUrl(url));
       expect(error.message).toBe('Invalid credential');
     });
+
+    it('Expect check if password is required', async () => {
+      const url =
```

Comment:
are these requests mocked? if not they should be, cant guarantee this url will exist in future and itd break the test